tool/template-calculator.html

```diff
@@ -16,35 +16,38 @@
 {% block breadcrumbs %}{% endblock %}
 
 {% block main %}
+<!-- <div id="fb-root"></div>
+<script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v8.0" nonce="IhVJTgzU"></script> -->
+
 <div class="fullwidth-bg">
-  <div class="container">
+  <div class="container mt-30">
 
     <section class="row mb-49">
       <h6 class="section-marker">{% trans %}Sustainability{% endtrans %}</h6>
-      <div class="col-sm-4 d-sm-none d-md-block sustainability-bg">
-        &nbsp;
+      <div class="col-sm-4 d-sm-none d-md-block">
+        <img class="mw-100" src="./img/green/green-graphic.png">
```

Comment:
For decorative images such as this one, we should always [provide empty alt text so that screen readers do not read the file name aloud](https://www.w3.org/WAI/tutorials/images/decorative/).

(There are many other cases of this throughout the new pages; we should comb through and fix them at some point.)

Reply:
Sounds good, I can take a look at that